{{cookiecutter.service_name}}/service.py

```diff
@@ -516,6 +516,30 @@ def {{cookiecutter.workflow_id |replace("-", "_")  }}(conf, inputs, outputs): #
 
         exit_status = runner.execute()
 
+        # Delete the temporary aws-credentials PVC
+        logger.info("Delete the temporary aws-credentials PVC")
+        # Load kubeconfig
+        config.load_incluster_config()
```

Comment:
Do we need to load kubeconfig again? As it was loaded before execution, line 465